client/src/pages/home/Home.css

```diff
@@ -1,8 +1,10 @@
 #root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 3px;
   text-align: center;
+  top: 0;
```

Comment:
💭Don't think we want to use `top: 0`. I imagine this is to get your navbar to the top of the screen but not a maintainable solution. Rather keeping the navbar to the top of the screen should be done within the shell component